app/controllers/api/v1/basic_auth_api/protocol_subscriptions_controller.rb

```diff
@@ -43,11 +43,25 @@ def destroy
           destroyed
         end
 
+        # This updates the protocol subscription. Only works if the external_identifier is given.
+        # We rescheduel the responses with a date that is in the future, for the same reasoning
+        # as is explained in reschedule_responses.rb.
+        def update
+          res = @protocol_subscription.update(protocol_subscription_update_params)
+          if res
```

Comment:
can't you do `return unprocessable_entity(@protocol_subscription.errors) if res.nil?`

Reply:
Sure can.

Reply:
Fixed.